sdk/core/core-lro/CHANGELOG.md

```diff
@@ -1,7 +1,8 @@
 # Release History
 
-## 1.0.5 (Unreleased)
+## 1.0.5 (2021-04-12)
 
+- Bug fix: Update compiled artifacts to use es6 instead of cjs modules.
```

Comment:
Maybe some nod that nothing actually changed in the code? e.g.

```suggestion
- No functionality changes from 1.0.4. This release is to correct an issue where 1.0.4 shipped with modules in the wrong format (cjs instead of es6.)
```